renderer.php

```diff
@@ -53,8 +53,19 @@ public function head_code(question_attempt $qa) {
      * @throws coding_exception
      */
     public function formulation_and_controls(question_attempt $qa, question_display_options $options): string {
+        global $DB;
         $question = $qa->get_question();
         assert($question instanceof qtype_questionpy_question);
+
+        // Check if we are in a quiz context.
+        if ($this->page->context->contextlevel == CONTEXT_MODULE && $this->page->cm->modname == 'quiz') {
+            $quiz = $DB->get_record('quiz', ['id' => $this->page->cm->instance]);
```

Comment:
Die Information steckt doch bereits in `$question->shuffleanswers`!?

Reply:
Wird das `$question->shuffleanswers` denn automatisch überschrieben, wenn man in den Quiz Settings > Question Behaviour > Shuffle within questions ändert, geändert?